source/extensions/io_socket/buffered_io_socket/user_space_file_event_impl.cc

```diff
@@ -13,25 +13,22 @@ namespace BufferedIoSocket {
 
 UserSpaceFileEventImpl::UserSpaceFileEventImpl(Event::Dispatcher& dispatcher, Event::FileReadyCb cb,
                                                uint32_t events, ReadWritable& io_source)
-    : schedulable_(dispatcher.createSchedulableCallback([this]() { cb_(); })), cb_([this, cb]() {
+    : schedulable_(dispatcher.createSchedulableCallback([this, cb]() {
         auto ephemeral_events = event_listener_.getAndClearEphemeralEvents();
         ENVOY_LOG(trace, "User space event {} invokes callbacks on events = {}",
                   static_cast<void*>(this), ephemeral_events);
         cb(ephemeral_events);
-      }),
+      })),
       io_source_(io_source) {
   setEnabled(events);
 }
 
-void EventListenerImpl::onEventEnabled(uint32_t) {
+void EventListenerImpl::clearEphemeralEvents(uint32_t) {
```

Comment:
Remove argument to clearEphemeralEvents()

Reply:
done